docs/pt/docs/fastapi-people.md

```diff
@@ -0,0 +1,137 @@
+# FastAPI Pessoal
+
+FastAPI has an amazing community that welcomes people from all backgrounds.
+FastAPI tem uma comunidade incrível que recebe pessoas de todas as origens.
+
+## Criador - Mantenedor
+
+Hey! 👋
+
+Esse sou eu:
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.maintainers %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Respostas: {{ user.answers }}</div><div class="count">Pull Requests: {{ user.prs }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+Eu sou o criador e mantenedor do **FastAPI**. Você pode ler mais sobre isso em [Ajuda FastAPI - Obtenha ajuda - Conecte-se com o autor](help-fastapi.md#connect-with-the-author){.internal-link target=_blank}.
+
+...Mas aqui eu quero mostrar a você a comunidade.
+
+---
+
+**FastAPI** recebe muito apoio da comunidade. E quero destacar suas contribuições.
+
+Estas são as pessoas que:
+
+* [Ajude outras pessoas com problemas (perguntas) no GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}.
+* [Crie Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}.
+* Revise Pull Requests, [especially important for translations](contributing.md#translations){.internal-link target=_blank}.
+
+Uma salva de palmas para eles. 👏 🙇
+
+## Usuários mais ativos no mês passado
+
+Estes são os usuários que mais têm [ajudado outras pessoas com problemas (dúvidas) no GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} durante o ultimo mês. ☕
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.last_month_active %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Problemas respondidos: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Especialistas
+
+Aqui estão os especialistas **FastAPI**. 🤓
+
+ 
+Estes são os usuários que mais [ajudaram outras pessoas com problemas (perguntas) no GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} through *all time*.
+
+They have proven to be experts by helping many others. ✨
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.experts %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Issues replied: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Top Contributors
```

Comment:
This is not translated...

Reply:
i will commit the end of translations, sorry

Reply:
No need for apologies :smile: